README.md

```diff
@@ -22,6 +22,8 @@ In order to interact with the MoJ Cloud Platform there are certain environment v
 - SLACK_WEBHOOK
 - SMOKE_TEST_USER
 - SMOKE_TEST_PASSWORD
+- NEW_RUNNER_USER
+- NEW_RUNNER_PASSWORD
```

Comment:
sorry to be picky, but can we specify it's for acceptance tests in the name please 🙇

Reply:
Done!